core/src/main/scala/stryker4s/extension/TreeExtensions.scala

```diff
@@ -98,4 +98,15 @@ object TreeExtensions {
     final def isIn[T <: Tree](implicit classTag: ClassTag[T]): Boolean =
       mapParent[T, Boolean](thisTree, _ => true, false)
   }
+
+  implicit class IsConditionOfIf(thisTree: Tree) {
```

Comment:
In my opinion this is fine to keep in the `ConditionalExpression` class. I don't see us using it anywhere else, and there already is a bit of tree logic there.